boards/BMS/Src/Io/Io_LTC6813/Io_SharedLTC6813.c

```diff
@@ -233,48 +242,48 @@ void Io_NewLTC6813_InitSpiHandle(SPI_HandleTypeDef *spi_handle)
         spi_handle, SPI2_NSS_GPIO_Port, SPI2_NSS_Pin, SPI_TIMEOUT_MS);
 }
 
-uint16_t Io_NewLTC6813_CalculateRegGroupPec15(uint8_t *data_buffer)
+uint16_t Io_SharedLTC6813_CalculateRegGroupPec15(uint8_t *data_buffer)
 {
-    return Io_NewLTC6813_CalculatePec15(
+    return Io_SharedLTC6813_CalculatePec15(
         data_buffer, NUM_REG_GROUP_PAYLOAD_BYTES);
 }
 
-void Io_NewLTC6813_PackPec15(uint8_t *tx_data, uint8_t num_bytes)
+void Io_SharedLTC6813_PackPec15(uint8_t *tx_data, uint8_t num_bytes)
 {
     const uint16_t cfg_reg_a_pec15 =
-        Io_NewLTC6813_CalculatePec15(tx_data, num_bytes);
+        Io_SharedLTC6813_CalculatePec15(tx_data, num_bytes);
 
     // Pack the PEC15 byte into tx_data in big endian format
     *((uint16_t *)(tx_data + num_bytes)) =
         CHANGE_WORD_ENDIANNESS(cfg_reg_a_pec15);
 }
 
-void Io_NewLTC6813_PrepareCmd(uint16_t *tx_cmd, uint16_t cmd)
+void Io_SharedLTC6813_PrepareCmd(uint16_t *tx_cmd, uint16_t cmd)
```

Comment:
What do you think of renaming it to `Io_SharedLTC6813_PrepareCmdWithPEC15`

Reply:
this isnt just preparing with pec15 though. its changing the cmd endianness and then doing the pec15 check.